### DISCUSSION TITLE
Plugin approval

### DISCUSSION THREAD

**Discussion by NyakudyaA** [stangandaho/Extractor#3] [General]
Please remove https://github.com/stangandaho/Extractor/tree/main/__pycache__, It's auto-generated on install

**Comment by stangandaho** [comment]
I just remove __pycache__ directory

On Mon, 13 Mar 2023 at 12:10, mazano ***@***.***> wrote:

> Please remove https://github.com/stangandaho/Extractor/tree/main/__pycache__,
> It's auto-generated on install
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/stangandaho/Extractor/issues/1>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN5EECPBAZ26GWU4CE6NR2TW336BLANCNFSM6AAAAAAVY5OHKA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

*Stanislas Mahussi GANDAHO*,

Ecological and Environmental Data Scientist

Wildlife Officer

Co-founder of Wlenwin <https://wlenwin.com>

Executive Secretary at ReAgro <https://re-agro.org/>

Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
<https://web.facebook.com/stangandaho/>, Instagram
<https://www.instagram.com/stangandaho/>, WhatsApp
<https://wa.me/+22995082578>

**Comment by stangandaho** [comment]
I noticed that the plugin icon does not appear correctly. It was the
default 'Plugin builder' icon that was showing up. I just fixed it. How to
publish this without creating a new version for the plugin?

On Mon, 13 Mar 2023 at 15:48, Stanislas Mahussi GANDAHO <
***@***.***> wrote:

> Please approve the last version released. I improve the plugin to work
> without external libraries.
> Thank you
>
> On Mon, 13 Mar 2023 at 13:28, Stanislas Mahussi GANDAHO <
> ***@***.***> wrote:
>
>> I just remove __pycache__ directory
>>
>> On Mon, 13 Mar 2023 at 12:10, mazano ***@***.***> wrote:
>>
>>> Please remove
>>> https://github.com/stangandaho/Extractor/tree/main/__pycache__, It's
>>> auto-generated on install
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/stangandaho/Extractor/issues/1>, or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AN5EECPBAZ26GWU4CE6NR2TW336BLANCNFSM6AAAAAAVY5OHKA>
>>> .
>>> You are receiving this because you are subscribed to this thread.Message
>>> ID: ***@***.***>
>>>
>>
>>
>> --
>>
>> *Stanislas Mahussi GANDAHO*,
>>
>> Ecological and Environmental Data Scientist
>>
>> Wildlife Officer
>>
>> Co-founder of Wlenwin <https://wlenwin.com>
>>
>> Executive Secretary at ReAgro <https://re-agro.org/>
>>
>> Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
>> <https://web.facebook.com/stangandaho/>, Instagram
>> <https://www.instagram.com/stangandaho/>, WhatsApp
>> <https://wa.me/+22995082578>
>>
>
>
> --
>
> *Stanislas Mahussi GANDAHO*,
>
> Ecological and Environmental Data Scientist
>
> Wildlife Officer
>
> Co-founder of Wlenwin <https://wlenwin.com>
>
> Executive Secretary at ReAgro <https://re-agro.org/>
>
> Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
> <https://web.facebook.com/stangandaho/>, Instagram
> <https://www.instagram.com/stangandaho/>, WhatsApp
> <https://wa.me/+22995082578>
>


-- 

*Stanislas Mahussi GANDAHO*,

Ecological and Environmental Data Scientist

Wildlife Officer

Co-founder of Wlenwin <https://wlenwin.com>

Executive Secretary at ReAgro <https://re-agro.org/>

Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
<https://web.facebook.com/stangandaho/>, Instagram
<https://www.instagram.com/stangandaho/>, WhatsApp
<https://wa.me/+22995082578>

**Comment by stangandaho** [comment]
If there is no possibility, you would like to approve the released version.
Thank you.

On Mon, 13 Mar 2023 at 19:21, Stanislas Mahussi GANDAHO <
***@***.***> wrote:

> I noticed that the plugin icon does not appear correctly. It was the
> default 'Plugin builder' icon that was showing up. I just fixed it. How to
> publish this without creating a new version for the plugin?
>
> On Mon, 13 Mar 2023 at 15:48, Stanislas Mahussi GANDAHO <
> ***@***.***> wrote:
>
>> Please approve the last version released. I improve the plugin to work
>> without external libraries.
>> Thank you
>>
>> On Mon, 13 Mar 2023 at 13:28, Stanislas Mahussi GANDAHO <
>> ***@***.***> wrote:
>>
>>> I just remove __pycache__ directory
>>>
>>> On Mon, 13 Mar 2023 at 12:10, mazano ***@***.***> wrote:
>>>
>>>> Please remove
>>>> https://github.com/stangandaho/Extractor/tree/main/__pycache__, It's
>>>> auto-generated on install
>>>>
>>>> —
>>>> Reply to this email directly, view it on GitHub
>>>> <https://github.com/stangandaho/Extractor/issues/1>, or unsubscribe
>>>> <https://github.com/notifications/unsubscribe-auth/AN5EECPBAZ26GWU4CE6NR2TW336BLANCNFSM6AAAAAAVY5OHKA>
>>>> .
>>>> You are receiving this because you are subscribed to this thread.Message
>>>> ID: ***@***.***>
>>>>
>>>
>>>
>>> --
>>>
>>> *Stanislas Mahussi GANDAHO*,
>>>
>>> Ecological and Environmental Data Scientist
>>>
>>> Wildlife Officer
>>>
>>> Co-founder of Wlenwin <https://wlenwin.com>
>>>
>>> Executive Secretary at ReAgro <https://re-agro.org/>
>>>
>>> Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
>>> <https://web.facebook.com/stangandaho/>, Instagram
>>> <https://www.instagram.com/stangandaho/>, WhatsApp
>>> <https://wa.me/+22995082578>
>>>
>>
>>
>> --
>>
>> *Stanislas Mahussi GANDAHO*,
>>
>> Ecological and Environmental Data Scientist
>>
>> Wildlife Officer
>>
>> Co-founder of Wlenwin <https://wlenwin.com>
>>
>> Executive Secretary at ReAgro <https://re-agro.org/>
>>
>> Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
>> <https://web.facebook.com/stangandaho/>, Instagram
>> <https://www.instagram.com/stangandaho/>, WhatsApp
>> <https://wa.me/+22995082578>
>>
>
>
> --
>
> *Stanislas Mahussi GANDAHO*,
>
> Ecological and Environmental Data Scientist
>
> Wildlife Officer
>
> Co-founder of Wlenwin <https://wlenwin.com>
>
> Executive Secretary at ReAgro <https://re-agro.org/>
>
> Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
> <https://web.facebook.com/stangandaho/>, Instagram
> <https://www.instagram.com/stangandaho/>, WhatsApp
> <https://wa.me/+22995082578>
>


-- 

*Stanislas Mahussi GANDAHO*,

Ecological and Environmental Data Scientist

Wildlife Officer

Co-founder of Wlenwin <https://wlenwin.com>

Executive Secretary at ReAgro <https://re-agro.org/>

Social networks: Twitter <https://twitter.com/stangandaho>, Facebook
<https://web.facebook.com/stangandaho/>, Instagram
<https://www.instagram.com/stangandaho/>, WhatsApp
<https://wa.me/+22995082578>

**Comment by NyakudyaA** [comment]
Hi @stangandaho You need to upload a new version or replace the current latest one. Once a plugin is uploaded an email is sent out to notify approvers. No need to ping me each time you fix something.
